extensions/cuid-generator/package.json

```
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "cuid-generator",
+  "title": "CUID Generator",
+  "description": "A quick way to generate CUIDs",
+  "icon": "command-icon.png",
+  "author": "dgrcode",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "generate",
+      "title": "Generate cuid",
```

Since `CUID Generator` is already the extension's title and will thus show up as the subtitle, there's no need to repeat it in the command's title. Instead, you could change the command to something like `Copy CUID`.

Instead of just copying to the clipboard, you could provide a preference to allow directly [pasting](https://developers.raycast.com/api-reference/clipboard#clipboard.paste) to the frontmost application. You can find an example of it [here](https://github.com/raycast/extensions/blob/main/extensions/cpf-cnpj-generator/package.json#L29-L36) 😸 


```suggestion
      "title": "Copy CUID",
```